### PR TITLE
Initial smatr-ads-net

### PR DESCRIPTION
**Pull request by approslab**

### CODE REVIEW

**Comment by nainar** [comment]
I am not the correct reviewer for Ads related code. @powerivq is more appropriate. Unassigning myself.

**Comment by powerivq** [comment]
@nainar adding you back since ads do not own the extensions/ folder, and we need your owner approval to create a new extension.

**Comment by kristoferbaxter** [comment]
@powerivq Can you also review? Your approval can be used as the signal for @nainar.

**Comment by powerivq** [comment]
@kristoferbaxter Yeah, I am.

File: extensions/amp-ad-network-smartads-impl/0.1/smartads-a4a-config.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * @param {!Window} win
+ * @param {!Element} element
+ * @param {boolean} useRemoteHtml
+ * @return {boolean}
+ */
+export function smartAdsIsA4AEnabled(win, element, useRemoteHtml) {
+  let src;
+  return (
+    !useRemoteHtml &&
+    !!(src = element.getAttribute('src')) &&
```
Comment (powerivq) [inline extensions/amp-ad-network-smartads-impl/0.1/smartads-a4a-config.js]: Is `=` intended?

Comment (approslab) [reply extensions/amp-ad-network-smartads-impl/0.1/smartads-a4a-config.js]: I don 't quite understand

Comment (powerivq) [reply extensions/amp-ad-network-smartads-impl/0.1/smartads-a4a-config.js]: Can this assignment be moved to below (line 32)? The logic here is a bit difficult to understand.

```
let src = element.getAttribute('src');
return (
    !useRemoteHtml &&
    !!src &&
    ....
```

Comment (approslab) [reply extensions/amp-ad-network-smartads-impl/0.1/smartads-a4a-config.js]: good idea

**Comment by kristoferbaxter** [comment]
@powerivq – Looks like a `e2e` failure within `inabox`.

https://travis-ci.org/github/ampproject/amphtml/jobs/666963627?utm_medium=notification&utm_source=github_status

**Pull request merged**